README.md

```diff
@@ -163,3 +163,35 @@ web: uvicorn main:app --host=0.0.0.0 --port=${PORT:-5000}
 </table>
 
 </details>
+
+**To Run application using Docker**
```

Comment:
This explanation is really cool. 

But can you add only 

<pre>
### Local Run

```
docker-compose up --build -d
```

http://localhost:8000/docs

</pre>

between Development and Deployment parts?

Reply:
@sadikkuzu I can remove this now submit it now if you say but the windows part , I will not be able to do